_config.yml

```diff
@@ -40,6 +40,7 @@ exclude:
   - Rakefile
   - Gemfile
   - Gemfile.lock
+  - gems/
```

Comment:
If I do `bundle install` on its own it errors that it can't install to the (global install) location. I tend to work around that using `--path gems` (as I noted in the srobo/website.git README). If there's a way to configure `bundle install` to use a per-user install I'd be happy to do that, but having a virtualenv-like per-project setup seems more sensible (as it allows different projects to have different requirements).

Speaking of READMEs, @jimmythompson I don't suppose you feel like copying over (most of) the README from the website repo so that there's a handy getting started guide in this repo as well? Alternatively, a link from one to the other might be a useful thing to help those who are not familiar with ruby/jekyll/bundle/etc..